dbt-cta/monitoring/models/1_intermediate/composer_logs/_int_composer_logs__models.yml

```diff
@@ -36,6 +36,45 @@ models:
       - name: data_interval_start
       - name: data_interval_end
       - name: dag_hash
+  
+  - name: int_group_scheduler_logs_by_dag
+    description: >
+      This model takes our parsed scheduler logs table and groups by dag run (dag_id and
+      run_id). When tasks within a given dag run get re-run (e.g. a task fails and is
+      re-run manually from the frontend), the scheduler logs a new entry for the dag run
+      when it finishes subsequent times.
+      
+      To handle this, we group all those entries together, and aggregate the fields
+      appropriately. We take the min() of start times, the max() of end times, the sum()
+      of durations, and a handful of other fields, including `state` from the most
+      recent log entry.
+    columns:
+      - name: dag_id
+        tests: 
+          - not_null
+      - name: run_id
+        tests:
+          - not_null
+      - name: insert_id
+      - name: log_timestamp
+      - name: received_timestamp
+      - name: project_id
+        tests: 
+          - not_null
+      - name: environment_name
+      - name: execution_date
+      - name: run_start_date
+      - name: run_end_date
+      - name: run_duration
+      - name: state
+        tests:
+          - accepted_values:
+              values: ['success', 'failed']
+      - name: external_trigger
+      - name: run_type
+      - name: data_interval_start
```

Comment:
QQ - what are these `data_interval` fields about? Can we add a description to one of them to explain?